### ISSUE TITLE
How do i delete loop?

### ISSUE THREAD

**Issue by Harvie** [free-wheeling/freewheeling#13]
Sometimes i create loop that i don't like. From the wiki i did not understood how can i delete it (eg. to reuse the button it's attached to). How can i do it?

**Comment by bill-auger** [comment]
in freewheeling everything is specified in one, or some combination of, the XML configs under $HOME/.fweelin/ - they loosely correspond to the screen layouts seen in the &lt;TAB&gt; cycle - most every action can be assigned to any trigger in any layout config file (midi pedals/notes, pc-keyboard buttons, joystick buttons) - each screen layout has default triggers for the most important actions - for example, the default for the pc-keyboard layout is &lt;SPACE + a-z&gt; - i think that is the only default noted on the wiki - the wiki could use improvement - generally, hold the spacebar, and re-trigger whatever triggered the original recording (e.g. the "a" key, piano C3, midi note 64, the red dance-pad marking)

one needs to know only which layout is in use, tweak it for the situation, or add features such as a key-binding to that layout file, if it is not already there, or create a new custom layout guided by this wiki section and the events tables:

  https://github.com/free-wheeling/freewheeling/wiki/XML-Configuration-System

the same information is also in your $HOME dir
  ~/.fweelin/config-help.txt

regarding erasing loops, note specifically the examples in the '&lt;!-- Erase Loops --&gt;' sections in $HOME/.fweelin/pckeyboard.xml and ./.fweelin/midikeyboard.xml and
whichever layout file you are actually using

**Comment by Harvie** [comment]
> and to be simple and intuitive to use for musicians who are not so technically adept

That's what i am talking about. Diving in XML files is not exactly "intuitive to use for musicians who are not so technically adept".

BTW i've installed from ArchLinux package and `pckeyboard.xml` was by default disabled in `interfaces.xml`. As a newbie it took me some effort to get this working. In demo video you said it's enough to switch to the pckeyboard mode using TAB key, but it obviously does not work when it's disabled in config.

Now i've found that pckeyboard is also commented out in your git. Why is that? I think it's confusing for people who just want to try without connecting (or buying) MIDI hardware. I mean.. Almost everybody has keyboard right?

**Comment by bill-auger** [comment]
yep thats a good point - probably the pc-keyboard interface should be the default for fresh installs

BTW - the video you watched was probably the original author: JP Mercury

**Comment by bill-auger** [comment]
i should also add for completeness, that by default, loops can be deleted by holding the space-bar and right-clicking on the loop with the mouse

and that there is a help screen in the program ('/' slash key) that shows the current keyboard and mouse bindings, such as:
